prowler/providers/azure/services/defender/defender_client.py

```diff
@@ -1,4 +1,4 @@
-from prowler.providers.azure.lib.audit_info.audit_info import azure_audit_info
 from prowler.providers.azure.services.defender.defender_service import Defender
+from prowler.providers.common.common import global_provider
 
-defender_client = Defender(azure_audit_info)
+defender_client = Defender(global_provider)
```

Comment:
Why this is receiving something called `global_provider` instead of some `Azure....`?